### DISCUSSION TITLE
can  docker volume support to store in many diffrent directory?

### DISCUSSION THREAD

**Discussion by sungaok** [moby/moby#45336] [Q&A]
### Description

 docker volume default store in one system disk directory.
but we  found that cause disk consuming very fast.
so we must chage different disk to store this volume.
but after that chage, we found disk will be exhuased about one years later.
can docker support more volumes from different disk?
that we could not add extra bigger disk , and copy, and change.
evently we can use many disk to store  volume or volumes.

**Comment by sam-thibault** [comment]
I will move this to a discussion topic.
